src/factory/index.js

```diff
@@ -0,0 +1,3 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Not a fan of index.js files.  the coding pattern require(./factory).createService() elevates the "service factory" so a level above all the others and makes it inconsistent.  While it happens that in this case the service is composed of the other factory outputs that does not scale.  What if there were another level above that?  Would we add another directory with an index.js file?  Then the runtime object structure would be reflected in the file system, and the former could be a graph where the latter is a hierarchy (assuming no file system links).  This is all especially true if we move to a more modular provider structure.